integration-tests/chopsticks/src/tests/xcm/limitedReserveTransfer/config.ts

```diff
@@ -0,0 +1,293 @@
+import * as PolkadotChainConfigs from '../../../network/index.js'
+import { initialBalanceKILT, keysAlice, keysBob } from '../../../helper/utils.js'
+import * as SpiritnetConfig from '../../../network/spiritnet.js'
+import * as HydraDxConfig from '../../../network/hydraDx.js'
+import { tx, query } from '../../../helper/api.js'
+
+import type { ApiPromise } from '@polkadot/api'
+import type { BasicConfig, BasicXcmTestConfiguration, BasisTxContext } from '../types.js'
+import type { SubmittableExtrinsic } from '@polkadot/api/types'
+
+interface Config extends BasicConfig {
+	precision: bigint
+}
+
+interface Query {
```

Comment:
I think here it's ok to make it specific to only sender and receiver since we know we are dealing with two chains only. So this is ok in this context.